src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerMiddlewareImpl.cs

```diff
@@ -112,6 +112,18 @@ static async Task Awaited(ExceptionHandlerMiddlewareImpl middleware, HttpContext
 
     private async Task HandleException(HttpContext context, ExceptionDispatchInfo edi)
     {
+        if (edi.SourceException is OperationCanceledException && context.RequestAborted.IsCancellationRequested)
+        {
+            _logger.RequestAbortedException();
```

Comment:
Is it better to also pass the exception to the log here?
```suggestion
            _logger.RequestAbortedException(edi.SourceException);
```

Reply:
Yes I believe. @Tratcher  Please share your views.

Reply:
We get complaints from developers about noisy logs with long stack traces for known conditions they can't do anything about. Don't add the exception for now, we've handled it.